test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,28 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
+	NodeAffinity *nodeAffinity
+	InterPodAffinity *interPodAffinity
+}
+
+// interPodAffinity priority configuration details.
+type interPodAffinity struct {
```

Comment:
Here and in nodeAffinity - please don't inline parts of top level structures (LabelSelector here and NodeSelectorRequirement in NA). You can always overwrite values or sth in the code if needed. Or call it a "template" and in the comment say that 'values' are ignored.

PodAffinity will need namespaces as well, and IIUC it's not used yet, so please just delete it.